### PR TITLE
LibWeb: Use unsigned long long for ProgressEventInit

### PR DESCRIPTION
**Pull request by shannonbooth**
Noticed this low hanging fruit FIXME while investigating some other changes - trivial to fix now that the IDL generator supports this :^)

### CODE REVIEW

**Review by LucasChollet — COMMENTED** [review]
Small typo:
s/usigned/unsigned in the commit message

**Comment by shannonbooth** [comment]
> Small typo: s/usigned/unsigned in the commit message

whoops, thanks! fixed

File: Userland/Libraries/LibWeb/XHR/ProgressEvent.idl [hunk 1]
```diff
@@ -12,6 +12,6 @@ interface ProgressEvent : Event {
 
 dictionary ProgressEventInit : EventInit {
     boolean lengthComputable = false;
-    unsigned long loaded = 0;
-    unsigned long total = 0;
+    unsigned long long loaded = 0;
+    unsigned long long total = 0;
```
Comment (awesomekling) [inline Userland/Libraries/LibWeb/XHR/ProgressEvent.idl:16]: Can we actually instantiate `ProgressEventInit` with huge 64-bit int values from JavaScript though? :yakthonk:

Comment (shannonbooth) [reply Userland/Libraries/LibWeb/XHR/ProgressEvent.idl:16]: I believe so, unless I am mistaken: https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibWeb/WebIDL/AbstractOperations.cpp#L344

**Pull request merged**